### ISSUE TITLE
[BUG] Occlusion culling details/markings

### ISSUE THREAD

**Issue by Servomotor** [tsunamayo/Starship-EVO#2963]
### Short Description of the issue:
Some of the details are not rendering and look as if they disappear. Propeach said this may have to do with Occlusion culling. 
...

### Steps to reproduce the issue:
Not sure how to reproduce it. I just see it happen sometimes. 
...

### Environment
- Starship EVO version: 20w36b
- Graphics Card: RTX 2080

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_
You can see it happen on the ramp floor in this video at 1:57, I would start around 1:55. Just after 1:56 into 1:57 you will see the issue. 
https://www.youtube.com/watch?v=6K3lmiZhcjs

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
[Servo -Gobb's Oddity - Occlusion Detail issue.zip](https://github.com/tsunamayo/Starship-EVO/files/5220843/Servo.-Gobb.s.Oddity.-.Occlusion.Detail.issue.zip)

**Comment by tsunamayo** [comment]
Whoa, truly amazing ship! Was it a sticker or else? Thanks

**Comment by WILDCATreactor** [comment]
same problem trying to put a lot of decals on my little hoverjeep, after a certain point the decals apply and have hitboxes so i can delete them, but there seems to be some upper limit on the number of decals and/or stickers you can place on one entity before new appliques spazz out and become invisible

Steps to reproduce: apply a large number of decals/stickers until they apply but have no visual, delete "x+1" and reapply until invisibility reoccurs, number required seems to vary

**Comment by Servomotor** [comment]
@tsunamayo Thank you. I am really enjoying this ship. 
If you watch the video at 1:56 to 1:57 and look at the floor, you can see stickers disappear and then reappear.

**Comment by tsunamayo** [comment]
@WILDCATreactor yes there will definitely be a limit if you push too much, especially from afar.

**Comment by WILDCATreactor** [comment]
noted, I like metal bawxes anyways, so having excuses to use fewer detailing methods is fine in my book, though that does imply I'll have to rely more on greebling... drat
